src/sql/base/query/browser/untitledQueryEditorInput.ts

```diff
@@ -44,6 +48,26 @@ export class UntitledQueryEditorInput extends QueryEditorInput implements IUntit
 		return this.text.model.hasAssociatedFilePath;
 	}
 
+	override async save(group: GroupIdentifier, options?: ISaveOptions): Promise<IEditorInput | undefined> {
+		let preProcessed = await this.text.save(group, options);
```

Comment:
I'd name this something like `fileEditorInput` instead, "preprocessed" isn't really an accurate name here (there's nothing being "processed")